doc/experimental/phase1/upgrade-grafana.md

```diff
@@ -0,0 +1,223 @@
+```
+
+## Extract configuration information
+Extract some configuration information that will be required in the Helm overrides:
+```text
+GRAFANA_HOST=$(kubectl get ingress -n verrazzano-system vmi-system-grafana -o jsonpath='{.spec.rules[0].host}')
+```
+
+## Create Helm Overrides File
+
+Generate a Helm overrides file, based on Verrazzano's default install of Grafana:
+
+```text
+cat > overrides.yaml <<EOF
+nameOverride: system-grafana
+podLabels:
+  app: system-grafana
+deploymentStrategy:
+  type: Recreate
+affinity:
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - podAffinityTerm:
+        labelSelector:
+          matchLabels:
+            app: system-grafana
+        topologyKey: failure-domain.beta.kubernetes.io/zone
+      weight: 100
+admin:
+  existingSecret: grafana-admin
+  userKey: username
+  passwordKey: password
+grafana.ini:
+  server:
+    domain: ${GRAFANA_HOST}
+    root_url: https://${GRAFANA_HOST}
+    enable_gzip: true
+  users:
+    allow_sign_up: false
+    auto_assign_org: true
+    auto_assign_org_role: Viewer
+  auth:
+    disable_login_form: true
+    disable_signout_menu: true
+  auth.basic:
+    enabled: false
+  auth.anonymous:
+    enabled: false
+  auth.proxy:
+    enabled: true
+    header_name: X-WEBAUTH-USER
+    header_property: username
+    auto_sign_up: true
+env:
+  PROMETHEUS_TARGETS: http://vmi-system-prometheus:9090
+livenessProbe:
+  failureThreshold: 3
+  httpGet:
+    path: /api/health
+    port: 3000
+    scheme: HTTP
+  initialDelaySeconds: 15
+  periodSeconds: 20
+  successThreshold: 1
+  timeoutSeconds: 3
+readinessProbe:
+  failureThreshold: 3
+  httpGet:
+    path: /api/health
+    port: 3000
+    scheme: HTTP
+  initialDelaySeconds: 15
+  periodSeconds: 20
+  successThreshold: 1
+  timeoutSeconds: 3
+resources:
+  requests:
+    memory: 48Mi
+containerSecurityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+  privileged: false
+  runAsGroup: 472
+  runAsNonRoot: true
+  runAsUser: 472
+serviceAccount:
+  name: verrazzano-monitoring-operator
+securityContext:
+  seccompProfile:
+    type: RuntimeDefault
+persistence:
+  enabled: true
+  existingClaim: vmi-system-grafana
+extraVolumeMounts:
+  - name: dashboards-volume
+    mountPath: /etc/grafana/provisioning/dashboardjson
+extraConfigmapMounts:
+  - name: datasources-volume
+    mountPath: /etc/grafana/provisioning/datasources
+    configMap: vmi-system-datasource
+  - name: dashboards-provider-volume
+    mountPath: /etc/grafana/provisioning/dashboards
+    configMap: verrazzano-dashboard-provider
+extraContainers: |-
+  - name: k8s-sidecar
+    env:
+    - name: LABEL
+      value: grafana_dashboard
+    - name: LABEL_VALUE
+      value: "1"
+    - name: FOLDER
+      value: /etc/grafana/provisioning/dashboardjson
+    - name: NAMESPACE
+      value: ALL
+    image: ghcr.io/verrazzano/k8s-sidecar:v1.15.0-20230922083013-7adaf012
```

Comment:
is this image intentional?

Reply:
It is, but I need to remove ghcr.io.  Fixed.